feat: Add json_array_length UDF (#8602)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/scalar-functions.md

    `CREATE STREAM LOGS (LOG STRUCT<CLOUD STRING, APP STRING, INSTANCE INT>, ...) WITH (VALUE_FORMAT='JSON', ...)`

### `JSON_ARRAY_LENGTH`

Since: 0.25.0

```sql
JSON_ARRAY_LENGTH(json_string) -> Integer
```

Given a string, parses it as a JSON value and returns the length of the top-level array. Returns

> `NULL` if the string can't be interpreted as a JSON array, i.e., it is `NULL` or the JSON value is

Reviewer: ```suggestion
`NULL` if the string can't be interpreted as a JSON array, for example, when the string is `NULL` or the JSON value is
```
